dandi/metadata.py

```diff
@@ -449,18 +451,21 @@ def toContributor(value, contrib_type):
             # else:
             #    contrib["identifier"] = models.PropertyValue()
             del item["orcid"]
+        if "affiliation" in item:
+            item["affiliation"] = [models.Organization(**{"name": item["affiliation"]})]
```

Comment:
`models.Organization(**{"name": item["affiliation"]})` can be written as just `models.Organization(name=item["affiliation"])`.

Reply:
updated to reflect this.